### PR TITLE
2021 Sponsor updates pre-conference  :pencil:

### LINKED ISSUE #30: Placeholder: Update "Become a Sponsor" page
**Expected behavior:**

**Actual behavior:**

**Steps to repeat behavior:**

-
-

### PR DESCRIPTION
**Pull request by dawnwages**
Closes #30 

Changes proposed in this PR:

- [x] Update Sponsors page to "Announcing Soon" message
- [x] Update Become a Sponsor page to remove IRL components
- [x] ADDING LATER  - NO INFO ON PLATFORM YET - Add Sponsor options for online-only event : https://docs.google.com/spreadsheets/d/1oYpHCS0ZhtRH1QGnMWjPYhQj05fsCKNP2WHYQQH1LoU/edit?ts=60ba717a

### CODE REVIEW

File: _pages/sponsors.html [hunk 1]
```diff
@@ -20,11 +20,29 @@ <h1>{{ page.heading }}</h1>
...
+  <!--start spacer -->
+    <div class="row column v-pad-bottom">
+      <div class="col-12">
+
+      </div>
+    </div>
+  </div>
+  <!-- end spacer -->
 
+  <!--
```
Comment (dawnwages) [inline _pages/sponsors.html:45]: Add "Your Logo Here" language for one sponsor row instead of hiding all

**Comment by jefftriplett** [comment]
Quick overview looks good, but I know a lot of things are up in the air until we have a streaming platform.

**Comment by dawnwages** [comment]
Hi I think this is ready to be merged! :) I'll create a new PR when we have the platforms.

**Comment by KatherineMichel** [comment]
I will review and merge soon, probably after the meeting.

**Review by KatherineMichel — APPROVED** [review]

**Comment by KatherineMichel** [comment]
@dawnwages Looks good! I approved it. I'm not sure if sprints are happening though. Also, it should be changed to merge into dev branch. That might be why there are a lot of extra files in the PR.

**Review by katialira — APPROVED** [review]

**Pull request merged**